src/odemis/driver/test/emitter_test.py

```diff
@@ -73,61 +73,50 @@ def tearDown(self):
 
     def test_simple(self):
         # should start off
-        self.assertEqual(self.dev.power.value, 0)
+        self.assertEqual(self.dev.power.value, [0.0, 0.0, 0.0, 0.0])
 
-        self.assertEqual(len(self.dev.emissions.value), len(self.dev.spectra.value))
+        self.assertEqual(len(self.dev.power.value), len(self.dev.spectra.value))
 
         # turn on first source to 50%
-        self.dev.power.value = self.dev.power.range[1]
-        em = self.dev.emissions.value
-        em[0] = 0.5
-        self.dev.emissions.value = em
-        self.assertGreater(self.dev.emissions.value[0], 0)
-
-        # turn on second source to 90%
-        self.dev.power.value = self.dev.power.range[1]
-        em[0:2] = [0, 0.9]
-        self.dev.emissions.value = em
-        self.assertGreater(self.dev.emissions.value[1], 0)
+        self.dev.power.value[0] = self.dev.power.range[1][0] * 0.5
+        self.assertGreater(self.dev.power.value[0], 0)
+
+        # turn on second source (first channel) to 90%
```

Comment:
That's the "third" source. => update the comment.

Reply:
It's correct, second source, first channel -> multiplex power idx 2.